app.rb

```diff
@@ -0,0 +1,133 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+require './person'
+require './checks'
+
+class App
+  include Checks
+  attr_accessor :books, :people
+
+  def initialize
+    @people = []
+    @books = []
+    @book = nil
+    @person = nil
+    @rents = []
+  end
+
+  def run(choice)
+    list_books if choice == 1
+    list_people if choice == 2
+    create_person if choice == 3
+    create_book if choice == 4
+    create_rental if choice == 5
+    list_rental if choice == 6
```

Comment:
- [ optional ] You may consider using a switch statement or loop as you have done in the `create_person` method. This is good but I think using a loop or a switch statement would make your worker neater

Reply:
I implemented the `switch statement` :+1: thank you for your suggestion and your review :smile: